appveyor.yml

```diff
@@ -17,7 +17,12 @@ configuration:
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
+  - ps: cl.exe /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext sarif src\Sarif.FunctionalTests\ConverterTestData\PREfast\src\AdvancedAliasing\AA_IndexAlias.cpp; ls
+  - ps: Push-AppVeyorArtifact aa_indexalias.sarif
+  - ps: .\jsonparse.ps1 "aa_indexalias.sarif"  $env:APPVEYOR_REPO_PROVIDER $env:APPVEYOR_PROJECT_SLUG
```

Comment:
There are two issues with the name "jsonparse.ps1":

1. General: The name isn't informative.

    The name of anything -- a class, a method, a file -- should specifically say what the thing is or does. This script replaces file URIs with GitHub URIs, and the name doesn't convey that (in fact, the name suggests that the script is something it is not: a JSON parser).

2. Specific: The name doesn't follow PowerShell conventions.

    There are [naming conventions](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx) for Powershell cmdlets, and although AFAIK those guidelines don't apply to the names of Powershell _script files_, I like to follow them anyway.  Powershell commands names have the form "Verb-Noun", and there is a list of [approved verbs](https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx).

In this case, the best fit among the approved verbs appears to be "Convert", so how about: `Convert-FileUrisToGitHubUris.ps1`?